README.md

```diff
@@ -61,8 +63,34 @@ below.
 
     The default process will install ``libAcesContainer.so`` to ``/usr/local/lib``
     and a number of header files into ``/usr/local/include/aces``
-
-
+    
+    
+		
+               
+* Windows 
+
+	* x86	
+	    
+	    From the root source directory:
+
+			$ mkdir build && cd build
```

Comment:
The windows build instructions should include windows commands rather than *nix commands.